docs/contribution/30-overview/3-concepts/5-frontend-fragment.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Frontend Fragment
+---
+
+import OperationLink from "@site/src/components/OperationLink";
+
+The configuration of frontend fragments serves to integrate an extension’s frontend directly into defined [anchors](#anchors) within the mStudio.
+This is particularly useful when the extension’s frontend should be seamlessly integrated into mStudio to provide the feel of a native feature.
+
+Frontend fragments are configured on an extension.
+More detailed information can be found in the [Extension reference](../../../reference/extensions).
+
+Frontend fragments are embedded into mStudio via an iframe.
+This iframe is not rendered directly in the visible area of the page.
+Instead, the mStudio waits for communication via window messages that indicate which known components should be rendered.
+This approach ensures a consistent user experience and enables features such as overlays.
+
+Custom components and CSS cannot be used.
```

Comment:
why? That's a pretty severe restriction. Therefore, at least a justification should be provided.

Reply:
added an explanation